playbooks/common/openshift-master/config.yml

```diff
@@ -310,12 +310,19 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
+- name: Configure masters
   hosts: oo_masters_to_config
   any_errors_fatal: true
   serial: 1
   vars:
     sync_tmpdir: "{{ hostvars.localhost.g_master_mktemp.stdout }}"
+
+    # Do not pass a version if this is first master. (openshift_docker will sort it out)
+    # For subsequent masters we pass either pre-existing version for the master (if already installed),
+    # otherwise the first master version.
+    openshift_version: "{{ openshift.common.version if openshift.common.version is defined else oo_first_master.openshift.common.version.split('-')[0] }}"
+    when: inventory_hostname != groups.oo_first_master.0
```

Comment:
I think we're ok, I think I discovered this morning that the whole vars passing here is unnecessary, we sort it all out with the initialize_openshift_version.yml stuff, it wasn't even being used here.